src/utils/developerUI/__test__/developerUI.test.ts

```diff
@@ -1,10 +1,39 @@
 import {
+    createDeveloperUIInternalHref,
     createDeveloperUILinkWithNodeId,
     createPDiskDeveloperUILink,
     createVDiskDeveloperUILink,
 } from '../developerUI';
 
 describe('Developer UI links generators', () => {
+    describe('createDeveloperUILinkForCurrentHost', () => {
```

Comment:
Tested function is very simple, tests here are mostly to describe different possible cases (tests as docs) and needed to prevent bugs on functions updates